packages/amplify-go-function-runtime-provider/src/localinvoke.ts

```diff
@@ -26,7 +26,7 @@ const buildLocalInvoker = async (context: any) => {
     // Build localInvoker
     context.print.info('Local invoker binary was not found, building it...');
     executeCommand(['mod', 'tidy'], true, undefined, localInvokerDir);
-    executeCommand(['build', MAIN_SOURCE], true, undefined, localInvokerDir);
+    executeCommand(['build', '-o', 'bootstrap', MAIN_SOURCE], true, undefined, localInvokerDir);
```

Comment:
what are the new params hers for? '-o', 'bootstrap'

Reply:
nvm i see the changes in the migration guide

Reply:
`bootstrap` is required in the new runtime and the command is copied from the doc https://github.com/aws/aws-lambda-go